frontend/src/metabase/collections/components/ActionMenu/ActionMenu.unit.spec.tsx

```diff
@@ -1,78 +1,98 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import { renderWithProviders } from "__support__/ui";
+import { Collection, CollectionItem } from "metabase-types/api";
 import {
   createMockCollection,
   createMockCollectionItem,
 } from "metabase-types/api/mocks";
-import ActionMenu, { ActionMenuProps } from "./ActionMenu";
+import ActionMenu from "./ActionMenu";
+
+interface SetupOpts {
```

Comment:
Should we be using `opts` for options? I'm fine either way, but just want to know what I should be using next time.

However, I'm inclined to use full names almost all of the time. I think `opts` is one of the very few places that use abbreviations.

Reply:
I think we explicitly added `SetupOpts` to frontend docs